Tools/llm-tool/LLMTool.swift

```diff
@@ -127,7 +130,7 @@ struct SyncGenerator: AsyncParsableCommand {
         var printed = 0
 
         for token in TokenIterator(
-            prompt: MLXArray(promptTokens), model: model, temp: args.temperature)
+            prompt: MLXArray(promptTokens), model: model, temp: args.temperature, topP: args.topP)
```

Comment:
Looks great!  Please add the equivalent near line 210:

```swift
        let (task, channel) = generate(
            prompt: MLXArray(promptTokens), model: model, temp: args.temperature)
```

Reply:
Will update it today, sorry I was away for a while :p